aikau/src/main/resources/alfresco/renderers/CommentsList.js

```diff
@@ -49,6 +49,33 @@ define(["dojo/_base/declare",
        * @default [{i18nFile: "./i18n/CommentsList.properties"}]
        */
       i18nRequirements: [{i18nFile: "./i18n/CommentsList.properties"}],
+
+      /**
+       * Whether to use a full-screen dialog when adding comments.
+       * 
+       * @instance
+       * @type {boolean}
+       * @default
+       * @since 1.0.43
+       */
+      addCommentsFullScreen: false,
```

Comment:
So, are you suggesting a second property on CommentsList, or that this `addCommentsFullScreen` property should set both?